src/config.ts

```diff
@@ -161,7 +161,7 @@ export interface Config {
 
   // MARK-SPECIFIC CONFIGS
   /** Area-Specific Config */
-  area?: AreaConfig;
+  area?: MarkConfig;
```

Comment:
No but we dont want to define so many subversion of it.  If users really wanna specify it, good luck!